sriovnet_switchdev.go

```diff
@@ -183,6 +183,33 @@ func findNetdevWithPortNameCriteria(criteria func(string) bool) (string, error)
 	return "", fmt.Errorf("no representor matched criteria")
 }
 
+// SetVfRepresentorSmartNICMacAddress sets the given MAC address for a host VF identified by pfID and vfIndex
+func SetVfRepresentorSmartNICMacAddress(pfID, vfIndex string, mac net.HardwareAddr) error {
```

Comment:
There is already a method to get the Peer mac address of a representor netdev. see: `GetRepresentorPeerMacAddress`
maybe we should have `SetRepresentorPeerMacAddress`. this can first be implemented using sysfs, then later on move to devlink, as we would eventually use devlink to perform these types of operations in a standard way.

The flow to set MAC address would be:
1. get VF representor netdev on smart-NIC via `GetVfRepresentorSmartNIC`
2. set its peer netdev MAC address using `SetRepresentorPeerMacAddress`